active-mq/values.yaml

```
@@ -0,0 +1,190 @@
+#######################
+image:
+  repository: apache/activemq-artemis
+  # This sets the pull policy for images.
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 2.39.0
+
+# This is for the secretes for pulling an image from a private repository more information can be found here: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
+# This is to override the chart name.
+nameOverride: ""
```

YAMLlint (1.35.1)

[error] duplication of key "nameOverride" in mapping

(key-duplicates)